### ISSUE TITLE
Rule Proposal: warn about duplicates in switch cases

### ISSUE THREAD

**Issue by arturparkhisenko** [eslint/eslint#4558]
> The version of ESLint you are using (run eslint -v)

`v1.9.0`
> When the rules will warn. Include a description as well as sample code.

```javascript
var a = 'A';
switch (a) {
  case 'A':
    break;

  //notify/warn about that:
  case 'A':
    break;

  default:
}
```
> Whether the rule prevents an error or is stylistic.

It's can be warn :warning: .. or error (typo error).
> Why the rule should be in the core instead of creating a custom rule.

Yes if we have to use this rule in other cases of duplicates
> Are you willing to create the rule yourself?

Currently not enough an experience to create it. :persevere:

**Comment by eslintbot** [comment]
@arturparkhisenko Thanks for the issue! If you're reporting a bug, please be sure to include:

1. The version of ESLint you are using (run `eslint -v`)
2. What you did (the source code and ESLint configuration)
3. The actual ESLint output complete with numbers
4. What you expected to happen instead

Requesting a new rule? Please see [Proposing a New Rule](http://eslint.org/docs/developer-guide/contributing/new-rules) for instructions.

**Comment by lo1tuma** [comment]
There is already a rule for this: [`no-duplicate-case`](http://eslint.org/docs/rules/no-duplicate-case.html).

**Comment by arturparkhisenko** [comment]
@lo1tuma disabled by default? sorry, thanks, my bad :/
